app/controllers/auth_source_ldaps_controller.rb

```diff
@@ -22,8 +22,6 @@ def edit
   end
 
   def update
-    # remove from hash :account_password if blank?
-    params[:auth_source_ldap].except!(:account_password) if params[:auth_source_ldap][:account_password].blank?
```

Comment:
It looks like that:
![pass](https://cloud.githubusercontent.com/assets/11807069/9579676/07a76c10-4ffd-11e5-87eb-bdd1868170a3.png)
so unless the user click the change button, the password field is disabled